src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -38,16 +39,44 @@ internal class Utils
 
             if (platform == Platform.Windows)
             {
-                tracerProfiler32 = FileExists(Path.Combine(tracerHome, "win-x86", "Datadog.Trace.ClrProfiler.Native.dll"));
-                tracerProfiler64 = FileExists(Path.Combine(tracerHome, "win-x64", "Datadog.Trace.ClrProfiler.Native.dll"));
+                if (RuntimeInformation.OSArchitecture == Architecture.X64 || RuntimeInformation.OSArchitecture == Architecture.X86)
```

Comment:
Shouldn't all of these here and below check `RuntimeInformation.ProcessArchitecture`?
```suggestion
                if (RuntimeInformation.ProcessArchitecture== Architecture.X64 || RuntimeInformation.ProcessArchitecture== Architecture.X86)
```